modules/oc-mirror-operator-catalog-filtering.adoc

```diff
@@ -0,0 +1,271 @@
+
+a|Scenario 11
+
+[source,yaml]
+----
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages: 
+    - name: compliance-operator
+        channels
+          - name: stable
+            minVersion: 5.6.0
+----
+|Within the selected channel of that package, all version starting `minVersion` up to channel head (not relying of shortest path from upgrade graph).
+
+a|Scenario 12
+
+[source,yaml]
+----
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages: 
+    - name: compliance-operator
+        channels
+          - name: stable
+            maxVersion: 6.0.0
+----
+|Within the selected channel of that package, all versions up to `maxVersion` (not relying of shortest path from upgrade graph); Head of channel is not included, even if multiple channels are included in the filtering.
+
+a|Scenario 13
+
+[source,yaml]
+----
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages: 
+    - name: compliance-operator
+       channels
+          - name: stable
+            minVersion: 5.6.0
+            maxVersion: 6.0.0
+----
+|Within the selected channel of that package, all versions between `minVersion` and `maxVersion`, not relying of shortest path from upgrade graph; Head of channel is not included, even if multiple channels are included in the filtering.
+
+a|Scenario 14
+
+[source,yaml]
+----
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.14
+    packages:
+    - name: aws-load-balancer-operator
+      bundles:
+      - name: aws-load-balancer-operator.v1.1.0
+    - name: 3scale-operator
+      bundles:
+      - name: 3scale-operator.v0.10.0-mas
+----
+|Only the bundles specified for each package are included in the filtering.
+
+a|Scenario 15
+
+[source,yaml]
+----
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages: 
+    - name: compliance-operator
+        channels
+          - name: stable
+        minVersion: 5.6.0
+        maxVersion: 6.0.0
+----
+|Error: Filtering by channel and by package min or max version should not be allowed.
+
+a|Scenario 16
+
+[source,yaml]
+----
+mirror:
+  operators:
+   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages: 
+    - name: compliance-operator
+        channels
+          - name: stable
+        minVersion: 5.6.0
+        maxVersion: 6.0.0
+----
+|Error: Filtering using full:true and min or max version is not allowed.
+
+a|Scenario 17
+
+[source,yaml]
+----
+mirror:
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+      full: true
+    packages: 
+    - name: compliance-operator
+        channels
+          - name: stable
+            minVersion: 5.6.0
+            maxVersion: 6.0.0
+----
+|Error: Filtering using full:true and min or max version is not allowed.
+
+|===
```

Comment:
@sherine-k Do we need all these examples to be included ? is that really required ?